src/libstd/collections/hash/map.rs

```diff
@@ -148,14 +148,11 @@ use crate::sys;
 /// The easiest way to use `HashMap` with a custom key type is to derive [`Eq`] and [`Hash`].
 /// We must also derive [`PartialEq`].
 ///
-/// [`Eq`]: ../../std/cmp/trait.Eq.html
-/// [`Hash`]: ../../std/hash/trait.Hash.html
-/// [`PartialEq`]: ../../std/cmp/trait.PartialEq.html
-/// [`RefCell`]: ../../std/cell/struct.RefCell.html
-/// [`Cell`]: ../../std/cell/struct.Cell.html
-/// [`default`]: #method.default
-/// [`with_hasher`]: #method.with_hasher
-/// [`with_capacity_and_hasher`]: #method.with_capacity_and_hasher
+/// [`RefCell`]: crate::cell::RefCell
+/// [`Cell`]: crate::cell::Cell
+/// [`default`]: Default::default
```

Comment:
This now goes to the trait page, instead of the list of implementations. I know `HashMap::default` doesn't currently work, can we keep `#method.default` until then? We don't have the ambiguity of String::from because there can only be one Default impl.

Reply:
IMO the trait page is fine but I'd rather get rid of the #s and have links work over having them link to the trait impl instead of the trait. I don't see a difference in utility there

Reply:
The difference is that going to the trait takes you to a new page and away from the docs you were originally reading. `#method.default` loads  basically instantaneously.